### ISSUE TITLE
Crowdfunded DataBase

### ISSUE THREAD

**Issue by popovn** [lesspass/lesspass#764]
We dont need db of users but we need db of sites with restrictions on length and quality of the passwords 

When my password from lesspass does not fit on the site - it is very difficult to search for information on it - what kind of quality should the password be

If you collect these restrictions at the stage of specifying a domain from several people and they match - change  default user profile  so that the settings fall under the restrictions

In a strange way, sometimes there are restrictions on top (for example, there were a couple of times the site insisted that the password was too long, or cant use "@" in password)

**Comment by homeautomaton** [comment]
I came here to discuss this very same concept. I'm interested in converting all of my passwords over to be stateless, but the very first site I tried had restrictions that greatly interfered with the usability of lesspass. 

The existing settings for length, inclusion of letters, numbers and symbols are just not enough specificity. A crowd-sourced database would be a logical way to collect and share what we learn from a site's stated restrictions plus any trial-and-error it takes to determine anything unwritten.

Thoughts I've had for additional settings:

Avoid any adjacent repeats
Minimum number of lowercase
Minimum number of uppercase
Minimum number of numbers
Minimum number of symbols
Whitelist of symbols
Blacklist of symbols (using default set)

**Comment by homeautomaton** [comment]
@popovn See #534, which makes mention of this existing database: https://github.com/apple/password-manager-resources/blob/main/quirks/password-rules.json

**Comment by guillaumevincent** [comment]
Yes closing this one in favor of https://github.com/lesspass/lesspass/issues/534
Password stateless is great. Those stupid password rules website are using are painful. I don't know if LessPass should try to help them.
